src/ostree/ot-admin-builtin-status.c

```diff
@@ -152,6 +152,8 @@ deployment_print_status (OstreeSysroot *sysroot, OstreeRepo *repo, OstreeDeploym
 
       g_print ("%s", output_buffer->str);
     }
+#else
+  g_autofree char *remote = ostree_deployment_get_osname (deployment);
```

Comment:
No, the osname/stateroot is never right for this.  Let's just do instead:
`const char *remote = NULL;`
instead.

Reply:
Sounds good, this whole ostree_repo_signature_verify_commit_data path seems irrelevant with gpg anyway, setting this as null nips it in the bud alright.